src/components/text-editor/examples/text-editor-with-inline-images-file-storage.tsx

```diff
@@ -0,0 +1,118 @@
+import { Component, h, State, Host } from '@stencil/core';
+import {
+    ImageInserter,
+    FileInfo,
+    ImageInfo,
+    LimelTextEditorCustomEvent,
+    LimelCheckboxCustomEvent,
+} from '@limetech/lime-elements';
+/**
+ * Handling inline images (with external file storage)
+ *
+ * To allow users to paste images directly into the text editor, you can
+ * listen to the `imagePasted` event, which is triggered when an image file
+ * is pasted into the editor.
+ *
+ * The `imagePasted` event contains an `ImageInserter` object, which you can
+ * use to insert a thumbnail of the pasted image into the editor.
+ * After the thumbnail is inserted, you can upload the image to an external
+ * file storage and insert the src url of the uploaded image using the
+ * `insertImage` method.
+ *
+ * If the image upload fails, you can insert a failed thumbnail using the
+ * `insertFailedThumbnail` method.
+ */
+@Component({
+    tag: 'limel-example-text-editor-with-inline-images-file-storage',
+    shadow: true,
+})
+export class TextEditorWithInlineImagesExample {
+    @State()
+    private value = 'Copy an image file and paste it here.';
+
+    @State()
+    private uploadImageFails = false;
+
+    public render() {
+        return (
+            <Host>
+                <limel-text-editor
+                    value={this.value}
+                    onChange={this.handleChange}
+                    onImagePasted={this.handleImagePasted}
+                    onImageRemoved={this.handleImageRemoved}
+                />
+                <limel-checkbox
+                    label="Upload image fails - insert failed thumbnail"
+                    onChange={this.handleFailedThumbnailChange}
+                />
```

Comment:
I think it would be nice to add a `limel-example-value` element below each of these two examples, so you can see the result.

```suggestion
                />
                <limel-example-value value={this.value} />
```

Reply:
Come to think of it, in the case of the base64 example, the value might be waaaay long I guess, so maybe we shouldn't add it there…? We can at least add it here I think. This is where it's more interesting anyway, I think?

Reply:
Hehe yea I was about to write that. We did have the limel-example-value first but it didn't really work with base64 😄 

I can try to add it in this example and see how it works our 👍

Reply:
⚡ [6f0b8e6](https://github.com/Lundalogik/lime-elements/pull/3464/commits/6f0b8e690487e3dfe080ed97cfc6b08609fbf705)